test/middleware/PostQuery.test.ts

```diff
@@ -1,11 +1,11 @@
 import 'jest';
-import PostQuery from '../../src/middleware/PostQuery';
+import postQuery from '../../src/middleware/PostQuery';
+import { LogQueryData } from '../../@types/log';
 
 describe('Test API key header verification', () => {
     let mockURI: string;
     let mockProjectID: string;
-    let mockQueryData: QueryData;
-    let postQuery: PostQuery;
+    let mockQueryData: LogQueryData;
 
     beforeEach(() => {
```

Comment:
Maybe this is why one of the tests below is failing for me. The tests should be agnostic to my local setup or move these to the `.env` file so I can set up the test environment on my machine without overriding these values for everyone.

Do I need to be running the web app on on Port 3000 to run the tests?

Reply:
yes, these tests depend on running the webapp